internal/kibana/fleet.go

```diff
@@ -91,11 +93,11 @@ func (cfg FleetConfig) Flags() []string {
 
 	baseFlags := []string{"-e", "-v", "--force", "--insecure", "--enrollment-token=" + cfg.EnrollmentToken}
 	if common.AgentVersionBase == "8.0.0-SNAPSHOT" {
-		return append(baseFlags, "--url", fmt.Sprintf("http://%s@%s:%d", cfg.ElasticsearchCredentials, cfg.FleetServerURI, cfg.FleetServerPort))
+		return append(baseFlags, "--url", fmt.Sprintf("https://%s@%s:%d", cfg.ElasticsearchCredentials, cfg.FleetServerURI, cfg.FleetServerPort))
 	}
 
 	if cfg.ServerPolicyID != "" {
-		baseFlags = append(baseFlags, "--fleet-server-insecure-http", "--fleet-server", fmt.Sprintf("http://%s@%s:%d", cfg.ElasticsearchCredentials, cfg.ElasticsearchURI, cfg.ElasticsearchPort), "--fleet-server-host=http://0.0.0.0", "--fleet-server-policy", cfg.ServerPolicyID)
+		baseFlags = append(baseFlags, "--fleet-server-insecure-http", "--fleet-server", fmt.Sprintf("https://%s@%s:%d", cfg.ElasticsearchCredentials, cfg.ElasticsearchURI, cfg.ElasticsearchPort), "--fleet-server-host=http://0.0.0.0", "--fleet-server-policy", cfg.ServerPolicyID)
```

Comment:
@adam-stokes @blakerouse this set of flags is causing the failures for 7.x branch. Any help about what flags to use in that version of the code base?

Reply:
Shouldnt this be `fleet-server-es`?

Reply:
Changed in https://github.com/elastic/e2e-testing/pull/1104/commits/b09d53ae84970624f46a39558b68009921f0f822. It probably escaped from the original review